### DISCUSSION TITLE
This is too much work!

### DISCUSSION THREAD

**Discussion by otakususumu95** [vgmtrans/vgmtrans#506] [Q&A]
Throw in a DS file for some sequence. And as I needed much than files, stop the damaged stimulation and much more, just flat colors than that of boring black and white.

**Comment by furmilion** [comment]
This doesn't even make sense.

**Comment by Bonboon229** [comment]
What on earth is this word salad?

**Reply by admin** [reply]
> What on earth is this word salad?

Please don't raise noise. Activity results in notifications us maintiners have to get to.
Be it issues, discussions, PRs: comment only when you have insight or something to report

**Reply by mikelow** [reply]
Also, please be respectful.
